Documentation/Blazorise.Docs/Pages/Docs/Components/InputMasks/InputMaskPage.razor

```diff
@@ -26,8 +26,12 @@
     </UnorderedListItem>
 </UnorderedList>
 
+<DocsPageSubtitle>
+    Examples
+</DocsPageSubtitle>
+
 <DocsPageSection>
-    <DocsPageSectionHeader Title="Basic Example">
```

Comment:
We do use the Example suffix in more places, if we now don't like this, should it be changed widely?

Reply:
My bad, I was under the impression we used Example alot more.